### ISSUE TITLE
Wordpress "more" seperator has no efect while listing the articles

### ISSUE THREAD

**Issue by l3enito** [pojome/elementor#462]
the  seperator is ignored in a listing of articles,. This classical feature of WP Blog seems not to be implemented.

**Comment by joshmarom** [comment]
Thank you for drawing our attention to this issue. It is a pretty tricky subject.
Currently we do not intend to tackle this issue in the near future.

**Comment by l3enito** [comment]
So elementor gets a "THUMB DOWN" from me, because it breaks standard WP features like the mentioned "teasers (more tag)" and the developer are not willing to put (better: have) it on the roadmap because it is a "pretty tricky subject"! 
I am not saying that it is easy to solve but closing a bug issue does not make it better.

**Comment by joshmarom** [comment]
@l3enito after digging a bit more into the topic, we are going to take a closer look at it and see what we can do about it.

Please bare in mind that Elementor is an open source free plugin that was released just 3 months ago.
We are a small team working really hard on improving this product, fixing every reported bug and adding new features for users like you to use for free. 

One of the things we are working on now, is a major mobile update that will allow Elementor users to create HQ responsive designs.

Back to the "more" seperator issue, we will look into it and will update on this thread.

**Comment by l3enito** [comment]
Thanks! 
I am eagerly waiting for the next updates! Good luck!

**Comment by sfranchi** [comment]
I hope this can be fixed soon. Thanks to all of you.

**Comment by sfranchi** [comment]
Hi there, @joshmarom no news about this issue?

**Comment by joshmarom** [comment]
Sorry, but currently we still don't have a solution for this.

**Comment by sfranchi** [comment]
Geez… may I help? Something to investigate? I’m a programmer also, perhaps if I’ve an idea of what is producing this I could help on solving the problem.

______________________
Sandro Franchi
https://sandrofranchi.com
sandro.franchi@gmail.com
Cell: +56-9-4416-7040
Skype: sfranchi

**Comment by AnbertNL** [comment]
Hope for a fix soon. I was changing my pages and posts using the Elementor builder but i'm going to revert these changes and remove Elementor for nog. If this get fixed i'll look at it again. Great builder but this is a showstopper for me.

**Comment by sfranchi** [comment]
Can't believe nobody cares about this, I'll have to remove elementor from all my posts...

**Comment by agusmu** [comment]
My question is, is there any WordPress page builder that already supports "more" separator when using page builder on a "post"? 

If yes, please share it here, so Elementor team can get some ideas to handle it.

I agree with @joshmarom that this is very tricky subject.

Why is it tricky? Because most of WordPress page builder are "section" / "row"-based. 

So, the big question is WHERE will you put this "more" separator tag? 

Imagine if you put "more" separator inside a section, or inside first column on a section. if it works, then it will make the "teaser" layout broken.

I am thinking a solution for this, but the best idea from me is, we need to create a dedicated "post separator" section to separate teaser and full content. but it means that we will not use "more" separator tag. 

For now, I ask all of my users to use Elementor for any post type, except "post" post type. 

If they want to use Elementor for "post" post type, they can do it in one condition, they should show featured image + "the_excerpt", not "the_content", on archive/blog page.  It works for us now, everyone understand about it.

Also, please remember, on WordPress, we also have other tag, for example "nextpage" tag, to split full content to separate pages... It is also not supported by Elementor for same reason...

**Comment by sfranchi** [comment]
To have a "post separator" section would be just perfect and enough, is the same concept of the "anchor" one. Text-separator-text, ideal.

Thanks!

**Comment by WPDevHQ** [comment]
I don't seem to understand this issue or where the need for it is.

Elementor is a "Page Builder" and it builds that page by filtering the `the_content()` function by inserting sections, columns and custom content. At what point do you need the "more tag"? To break the page into sections or to fire a pagination type effect to the page?

Yes, you can use Elementor to also author a post as you would a page - the result is expected to be show on a"single post" view once gaian using the `the_content()` function. So the question again is, at what point on a single post view would you need the "more tag"?

If I understand correctly, the issue maybe on the "home/blog/archive"(not authored via the builder) pages - now, if those templates use `the_content()` then that is what you'll get, the full content and, if they use `the_excerpt()` then that is what you'll get. At this point I have to ask, how would/should Elementor or any other "page builder" account for the said content being output on other locations other than the page/post it was authored on?

As far as I can see, its the theme's responsibility to return either an excerpt or the full content.

Yes I understand the intent and need of the more tag to work but this is a page builder that filters and injects its content in to the said function. But what you're asking for is for the plugin to facilitate the breaking of the said content(comprising of sections) in multiple sections/pages where the WP_Editor is used and that does not compute with me since that content is meant to be output on a single page/post view!

Maybe someone can enlighten me on the dire need?

**Comment by l3enito** [comment]
>As far as I can see, its the theme's responsibility to return either an excerpt or the full content.
NOT REALLY.... 
"There are three different ways to show just an “introduction” to your post content."
Please have a look here:  http://buildwpyourself.com/wordpress-manual-excerpts-more-tag/

The   tag is a fundamental method in wordpress, but has a problem with Elementor:
"Unlike automatic excerpts, no HTML is stripped. "

Other editor software use custom tags to implement layout, elementor use HTML. Here might be the problem: Open HTML-tags would not be closed in the listing output. Every listing layout will break. 

A solution could be a parser that strips all elementor HTML-tags .... or a special “introduction” box that is only used for listing pages...

**Comment by WPDevHQ** [comment]
As stated, I understand the intent and perhaps the need.

But in the context of a "page" builder when does the need arise? Understanding this will help the team in working out a solution.

**Comment by sfranchi** [comment]
@WPDevHQ, If you're using a page builder for creating a post, you need to specify where the "read more" should happen in order to break the content in listings. So if the "read more" will not happen inside an editor section, a "read more" specific widget would be a nice to have in order to use Elementor in posts. If there is not a way to break the content where the author likes to, Elementor should be avoided in posts and reserved just for pages.

**Comment by WPDevHQ** [comment]
Or you can create a child theme and use `the_excerpt()` for the blog list page which works the same.

**Comment by sfranchi** [comment]
No @WPDevHQ, the_excerpt will not work with a manual break inside an Elementor text block. That's the issue, the "arbitrary" breaks, obviously common and part of the core functionality of WP.

**Comment by WPDevHQ** [comment]
Yes I understand that. But as explained by the team, it's a tricky issue to resolve in the realm of a "page builder". And as I understand it, that is the case for all page builders?

So until such time when the team can come up with a solution be it an improvised one, the immediate solution is to use `the_excerpt()` wouldn't you agree?

We can continue to rush the team and they could come up with a half baked solution, which won't help much. Or we can imrpovise for now and give the time to knock their heads together and come up with a solid solution :)

**Comment by sfranchi** [comment]
Yes, I do agree, I've already managed that on my sites @WPDevHQ. I'm not rushing the team at all, don't get me wrong, just trying to get a better product.

**Comment by WPDevHQ** [comment]
Cool.

From what I've seen from the team....Elementor can only get better. All in good time.

**Comment by sfranchi** [comment]
Agreed :)

**Comment by umanskiy** [comment]
What's in the Pro version is also such a problem?

**Comment by senseicorner** [comment]
There is a plugin that fully integrates with Visual Composers called Advanced post pagination. maybe elementor can look at something like that.  Not sure if it will help, but I much rather use elementor than Composer if I can help it. 

> My question is, is there any WordPress page builder that already supports "more" separator when using page builder on a "post"?

If yes, please share it here, so Elementor team can get some ideas to handle it.

I agree with @joshmarom that this is very tricky subject.

Why is it tricky? Because most of WordPress page builder are "section" / "row"-based.

So, the big question is WHERE will you put this "more" separator tag?

Imagine if you put "more" separator inside a section, or inside first column on a section. if it works, then it will make the "teaser" layout broken.

I am thinking a solution for this, but the best idea from me is, we need to create a dedicated "post separator" section to separate teaser and full content. but it means that we will not use "more" separator tag.

For now, I ask all of my users to use Elementor for any post type, except "post" post type.

If they want to use Elementor for "post" post type, they can do it in one condition, they should show featured image + "the_excerpt", not "the_content", on archive/blog page. It works for us now, everyone understand about it.

Also, please remember, on WordPress, we also have other tag, for example "nextpage" tag, to split full content to separate pages... It is also not supported by Elementor for same reason...

**Comment by senseicorner** [comment]
@umanskiy Currently the pro version has the same problem but their current updates give me hope still.

**Comment by l3enito** [comment]
What a about a a special elementor “introduction” box that is only used for listing of post types. 
- It should be a "texteditor box" as a meta information (you just have on instance on a page) 

In listings it will be shown only, because:
- it is the "first element" on the page 
-  Elementor sets the "more" function automatically to that box

And:
- it will not render on page views

That's it?

**Comment by senseicorner** [comment]
Personally I was hoping for a simple short code solution. They have a short code box in the pro version. They just need to make the page splitting work with it. But I am a beginner so I am not sure how doable that is.

**Comment by senseicorner** [comment]
from a customer point of veiw all web page builders focus on home, sales, about, ect. but How often do you build those pages. if they wanted to take the market by storm, then they should focus on content pages such as blog post. these we would use on at least a monthly and more likely a weekly basis. 

Building longer and more engaging content has become so important, but I feel none of these page builders are really looking at their market needs.

**Comment by stratboy** [comment]
The read more button should be implemented in Elementor's editors. Maybe with another name, just to not to conflict with the native wordpress one.

The purpose would not be the wordpress original one, since it would not have any practical usage. The real purpose, inside elementor is to give the user a way to break text inside a specific text box. So it not to be interpreted as a mean to break the entire page, but just a single text. So for example, one can then add a read more button that, once clicked, just shows the remaining text.

So to recap, it would be great if the elementor's version of wysiwyg editor had a custom button to break the text written in that specific editor. Just this.

**Comment by stratboy** [comment]
Also, the API should let ddevelopers to add new custom buttons to the wysiwyg control.

**Comment by roireshef** [comment]
So? Almost a year since the bug was opened. Any resolution on that? The "More" tag is a basic WP functionality. It just makes the Elementor unusable.

**Comment by sofitness** [comment]
Did you resolve the problem for adding a Read More button to Elementor ???
thanks

**Comment by stratboy** [comment]
@sofitness no, but please see my answer above, probably it can help.

**Comment by sofitness** [comment]
@stratboy  :-( thanks.....it's really a pitty..i do not understand why <!more.... do not work! 
Anyway, could you tell me what is the HR button ?? 
Sorry i am a debutant...and moreover...french. :-)

**Comment by stratboy** [comment]
The default more button I think it's quite difficult to implement and it's a bit too long to explain here, but just believe me.. I proposed an alternative, maybe they could implement it. In the meanwhile, you can use my code, and eventually modify it.

The text editor's HR button is the button you normally use to add HR elements.
From what you wrote, I can guess that maybe you do not know too much wordpress, nor html, and maybe nor php. If this is the case, I strongly recommend you to first study these things. Probably it could be a bit too early for you right know to code customisations like this, you could end up with more problems than before.

**Comment by webdevjeffus** [comment]
OK, here's why we need a read-more tag. Elementor now includes a really nice gallery feature, which I have used on my homepage. Now I need to include a different gallery in a post--but to use the SAME style gallery I used on my homepage, I have to edit the post with Elementor. This is a problem because the post, with it's featured image, gallery, and 400 words of text, is MUCH too long to appear on an index page. I need to insert a read-more tag to limit how much appears in any listing (Archive, Recent posts, main Blog page, etc.)

NOT including a read-more tag is not an option, but having to find, install and use a different gallery is reduntant, and gives the site an inconsistent look.

There has to be an elegant solution to this.

**Comment by fransstudio2** [comment]
Hey guys, 

i snooped around and see that the button is deliberately removed by elementor... for reason i dont know at the time being. Maybe it intersect with the Tiny MCE Javascript initialization?

On /wp-content/plugins/elementor/assets/js/editor.js line 9245 is it removed:

```
	// List of buttons to move {buttonToMove: afterButton}
	buttons: {
		moveToAdvanced: {
			blockquote: 'removeformat',
			alignleft: 'blockquote',
			aligncenter: 'alignleft',
			alignright: 'aligncenter'
		},
		moveToBasic: {},
		removeFromBasic: [ 'unlink', 'wp_more' ],
		removeFromAdvanced: []
	},
```

If you just change removeFromBasic to 

`		removeFromBasic: [ 'unlink' ],`

The button would show up - but I havent tested if tiny mce then would break or not. I know a friend that just "paste" the more tag in the code and it functions seamlessly.

**Comment by parfilov** [comment]
Maybe "Read more" widget like an "Anchor" that appears when editing post only?

**Comment by Fernsehstuhl** [comment]
I need the more tag too ... 

Any other idea?

**Comment by mr910** [comment]
Any update on this? I noticed pagination was briefly mentioned here and I'm having the same issue. I've got pagination enabled, but it doesn't change any of the post data when the next page is clicked on.

**Comment by otinoff** [comment]
This is problem for my site too.

**Comment by jezwebb** [comment]
Elementor does seem to show far too much text on blog category pages, and ignores the manually added <--more--> break. A work around I've found works is to add this to the bottom of your functions.php file.

function wpdocs_custom_excerpt_length( $length ) {    return 30;    }
add_filter( 'excerpt_length', 'wpdocs_custom_excerpt_length', 999 );

**Comment by fat32lobster** [comment]
Is this issue still in development? Have anyone found a good workaround?

**Comment by mitus66** [comment]
Hi, I can not apply a tag <!- -nextpage- ->  to divide the post into pages by html-code widget. Elementor created a pagination but has been duplicating the content before and after this tag.
I think the work of these tags ( <!- -more- ->, <!- -noteaser- -> and <!- -nextpage- -> ) is as natural as breathing.

**Comment by mitus66** [comment]
I'm surprised - almost 3 years nothing has been done to solve the problem

**Comment by joshmarom** [comment]
@mitus66 I'm assuming you meant "almost 2 years" :)

We did come up with a solution for this, it is currently under review.

**Comment by SaadMansuri** [comment]
Thanks , @joshmarom we are eagerly waiting for Pagination because I am having tough time to add customized pagination for my website(https://seleniumtpoint.com) default pagination for word press looks ugly .
Hence I am not using pagination currently  and user might have to scroll down a lot. on my website  Please help.

**Comment by SaadMansuri** [comment]
yes

**Comment by feznizzle** [comment]
I just wanted to pile on as yet another user who is *desperate* to get Pagination working with Elementor!

**@joshmarom**, you said:
> We did come up with a solution for this, it is currently under review.

I assume that means Pagination will soon work with Elementor? Any chance you could give us an update on rollout?

**Comment by DeanE1** [comment]
Hi, another frustrated user. I spent the last 4 days trying to get this to work. I'm currently writing a comparison post reviewing 14 cars and a need a way to separate each vehicle onto their own page. The use of Elementor has broken this basic WordPress function from working which I was not aware of. A fix would be greatly appreciated! I would even take a beta or temporary fix modifying code at this point if I had to.

**Comment by CH9il** [comment]
OMG Its been 2 years since this put up and no proper fix. Is this for real !!! 

I am wishing I didn't pay for elementor pro now !!! 

Sort it out or start stating the fact there is no help with on your website and you cant do it before someone pays money !!!!

**Comment by carasmo** [comment]
This is horrible. Standard stuff.

**Comment by patcascsaba** [comment]
We are eagerly waiting for this fix. Do you have an ETA of when can we expect it?

**Comment by mitus66** [comment]
Nothing has changed. The tag **<!- -nextpage- ->** is not working still. All pages are duplicated.

**Comment by tim777z** [comment]
The Elementor team, really need to pull up their socks. I didn't think pagination would be a big deal or issue, until my clients website became full of content, and users started complaining about "having to scroll too much", and the need for pagination to enhance usability on mobile devices and small screens...

**Comment by Yenuel-DA** [comment]
I found a temporary solution for most cases, using tabs. You just have to use {break-page} instead of  <!- -nextpage- ->.

the widget is attached and the code :arrow_forward: 

`	protected function render() {
		$settings = $this->get_settings_for_display();
		$subject=$settings['title'];
		//this is the hack replace </p> with a line break
		$subject=str_replace("</p>","\n",$subject);
		$separator = "\n";
		$line = strtok($subject, $separator);
		$page_number = 1;
	
		
		while ($line !== false) {
			if (strpos($line, '{break-page}') !== false) {
				//page break on the first line? nuts! but handling it anyway
				if($page_number==1){
					echo '<div class="title">';
					$classy="page-$page_number";
					echo "<div id='$page_number' class = ";
					echo "'page-$page_number paged shown-page'>";
					//let's remove the {break-page} from the line and print it
					echo str_replace("{break-page}","",$line);
					echo "</div>";
					echo "</div>";
					
				}
				
				else{
					//we know it's first page, so nothing to hide here
				}
				
				$page_number ++ ;
				
				
			}
			else{
				// and we didn't find {break-page}
				echo '<div class="title">';
				if($page_number==1){
					//it's the first page and there is no {break-page} on this line
					//so no need to perform str_replace
					$classy="page-$page_number";
					echo "<div id='$page_number' class = ";
					echo "'page-$page_number paged shown-page'>";
					echo $line;
					echo "</div>";
					
				}
				else{
					//it's Not the first page, so hide everything
					$classy="page-$page_number";
					echo "<div id='$page_number' class = ";
					echo "'page-$page_number paged hidden-page'>";
					echo $line;
					echo "</div>";
					
				}
				
				echo '</div>';
			}


		$line = strtok( $separator );
		
		}
		
		echo "<div class='page-links'> Pages:";
		$z=1;
		while ($z<= $page_number){
			if($z==1){
				$class="page-$z page-buttons activated";
			}
			else{
				$class="page-$z page-buttons";
			}
			
			$classy="page-$z";
			echo "<a id='page-$z' class='$class' onclick='openPage(event, $z)'>";
			echo $z ; 
			echo "</a>";
			$z++;
			
		}
		echo "</div>";
	
	  

		?>

<style>
	.hidden-page {
		display: none;
	}
	.shown-page{
		display:block;
	}

	.activated{
		background:gainsboro;
	}
	.page-buttons{
		padding:20px;
		margin-left:5px;
		color:black;
		text-align:center;
	}

</style>

<script>
function openPage(evt, pageClass) {
	var allPages = document.getElementsByClassName("paged");
    for (i = 0; i < allPages.length; i++) {
        allPages[i].style.display = "none";
    }

	var tablinks = document.getElementsByClassName("page-buttons");
    for (i = 0; i < tablinks.length; i++) {
        tablinks[i].className = tablinks[i].className.replace(" activated", "");
    }

	var currentPage = document.getElementById(pageClass);
	currentPage.style.display = "block";
	evt.currentTarget.className += " activated";
}
</script>

		
<?php			

	}
`

[custom-text-widget-elementor-basic.zip](https://github.com/pojome/elementor/files/2611942/custom-text-widget-elementor-basic.zip)

**Comment by Yenuel-DA** [comment]
here is a hack that works by using {break-page} instead of <!- -nextpage- -> inside a custom text editor widget. it's essentially using JS and CSS to hide/show individual tabs and dividing the text into smaller tabs inside a render function of the custom WYSIWYG text editor. Which can then be controlled by page buttons.

here is the code: 

```
	protected function render() {
		$settings = $this->get_settings_for_display();
		$subject=$settings['title'];
		//this is the hack replace </p> with a line break
		$subject=str_replace("</p>","\n",$subject);
		$separator = "\n";
		$line = strtok($subject, $separator);
		$page_number = 1;
		$new_page = false;
		
		while ($line !== false) {
			if (strpos($line, '{break-page}') !== false) {
				//page break on the first line? nuts! but handling it anyway
				if($page_number==1){
					echo '<div class="title">';
					$classy="page-$page_number";
					echo "<div id='$page_number' class = ";
					echo "'page-$page_number paged shown-page'>";
					//let's remove the {break-page} from the line and print it
					echo str_replace("{break-page}","",$line);
					echo "</div>";
					echo "</div>";
					
				}
				
				else{
					//we know it's first page, so nothing to hide here
				}
				
				$page_number ++ ;
				
				
			}
			else{
				// and we didn't find {break-page}
				echo '<div class="title">';
				if($page_number==1){
					//it's the first page and there is no {break-page} on this line
					//so no need to perform str_replace
					$classy="page-$page_number";
					echo "<div id='$page_number' class = ";
					echo "'page-$page_number paged shown-page'>";
					echo $line;
					echo "</div>";
					
				}
				else{
					//it's Not the first page, so hide everything
					$classy="page-$page_number";
					echo "<div id='$page_number' class = ";
					echo "'page-$page_number paged hidden-page'>";
					echo $line;
					echo "</div>";
					
				}
				
				echo '</div>';
			}


		$line = strtok( $separator );
		
		}
		
		echo "<div class='page-links'> Pages:";
		$z=1;
		while ($z<= $page_number){
			if($z==1){
				$class="page-$z page-buttons activated";
			}
			else{
				$class="page-$z page-buttons";
			}
			
			$classy="page-$z";
			echo "<a id='page-$z' class='$class' onclick='openPage(event, $z)'>";
			echo $z ; 
			echo "</a>";
			$z++;
			
		}
		echo "</div>";
	
	  

		?>

<style>
	.hidden-page {
		display: none;
	}
	.shown-page{
		display:block;
	}

	.activated{
		background:gainsboro;
	}
	.page-buttons{
		padding:20px;
		margin-left:5px;
		color:black;
		text-align:center;
	}

</style>

<script>
function openPage(evt, pageClass) {
	var allPages = document.getElementsByClassName("paged");
    for (i = 0; i < allPages.length; i++) {
        allPages[i].style.display = "none";
    }

	var tablinks = document.getElementsByClassName("page-buttons");
    for (i = 0; i < tablinks.length; i++) {
        tablinks[i].className = tablinks[i].className.replace(" activated", "");
    }

	var currentPage = document.getElementById(pageClass);
	currentPage.style.display = "block";
	evt.currentTarget.className += " activated";
}
</script>

		
<?php			

	}

```

**Comment by Yenuel-DA** [comment]
And here is the widget.

[custom-text-widget-elementor-basic.zip](https://github.com/pojome/elementor/files/2611947/custom-text-widget-elementor-basic.zip)

**Comment by Yenuel-DA** [comment]
@KingYes Cool. The team is working on it. 

@awwe-awwe   no need to use the widget since the elementor team is working on it. It was only meant to be a temporary hack.

**Comment by mrmoret** [comment]
Any ETA for this fix?

**Comment by Ioichka** [comment]
Just another user on the queue. I love using Elementor and its recent updates replaced a few of the plugins on the website, which is great. However, post pagination is extremely important for my blog as I have mile-long posts I need to split. Is there any update on the issue?

**Comment by jjisnow** [comment]
As another idea, you could always resort to creating a 'read more' button as in this link:

[https://www.w3schools.com/howto/howto_js_read_more.asp](w3schools-how-to-js)

**Comment by jprknight** [comment]
I would also like to be able to split some long posts. Adding another user requesting this feature.

**Comment by Todderz** [comment]
This is a critical problem for any serious blog and will force me to look for a different approach without Elementor. You really need to support . It is a very widely used technique and is massively important for fast page loading and preventing readers abandoning the page.

I understand it may be hard to solve but please understand there is significant demand for it. Thanks.

**Comment by mstudioIL** [comment]
I moving site that use Avada theme to other theme with Elementor and want to split the pages like the old site.
Avada has the option https://theme-fusion.com/documentation/fusion-builder/elements/next-page-element/.
How can I have with ?

**Comment by tmbmvp2180** [comment]
well what's it been 4 years and still not a fix....I am another elementor pro user who is now looking elsewhere....

**Comment by Nodws** [comment]
im form 2025 still no fix

**Comment by nisiwi** [comment]
+1 for pagination / page break support on pages.  This is an ideal way of breaking up long posts/pages into more desirable lengths for SEO/UX.

**Comment by carlosdeandrade** [comment]
has this really taken 4 years to get fixed?. I have a really long post that need to split into different pages but when activating the Elementor Editor all my  marks break...

Is anyone aware of any handy workaround please?

**Comment by njblue55** [comment]
How can this possibly be closed?? It's a critical function that impacts users and this has been open for over 4 years? I paid for Elementor Pro and have wasted several days looking for a workaround just to advance the page. Why should we need to jury rig something with additional code or search for a plug-in to do something as basic as paginating to the next page? I've opened a ticket days ago and the only response I've received is "We'll get back to you as soon as we can..." Seeing as how this thread is 4 1/2 years old I guess I shouldn't hold my breath.

**Comment by enricmm** [comment]
Me too I am a bit surprised this BASIC feature to the blog (to WP, in fact!) isn't available in Elementor... moreover after four years of this being discussed. In this day and age, lots of blogs have mile-long articles that need to be "paginated" in order to make them readable to users on mobile phones.

I am new at Elementor and I am also evaluating whether this can become a standard tool I use for my developments. Until now I had been using WpBakery which has something called "Pageable container" which performs the equivalent to . As per the above explanations in this thread, I can understand the underlying framework in Elementor works in a different way than WpBakery's but nonetheless, this functionality goes beyond technical considerations as it's a de-facto must for many bloggers, especially these days with the use of small-screen devices.

Thus, question for the developers: IS THERE ANY CLEAR INDICATION ON WHETHER THIS BASIC FEATURE will be implemented or not? And if so, when?

Thanks for your

**Comment by Alexanderipo** [comment]
@Yenuel-DA Please, how can I use the code you provided? Is it a plugin code?

**Comment by tim777z** [comment]
I Found a solution, there is a plugin for elementor called page break for
elementor. It paginates content using the <br> tag

**Comment by Alexanderipo** [comment]
@tim777z Please, where can I get the plugin? Can you provide a link to the plugin? I searched it online and nothing.

**Comment by tim777z** [comment]
https://www.wppagebuilders.com/add-pagination-in-elementor/
